### PR TITLE
[feature](multi-catalog) support oceanbase jdbc catalog and jdbc external table

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.39 seconds
 stream load tsv:          428 seconds loaded 74807831229 Bytes, about 166 MB/s
 stream load json:         25 seconds loaded 2358488459 Bytes, about 89 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230423101947_clickbench_pr_133319.html

**Comment by LemonLiTree** [comment]
please add doc

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
> please add doc

PTAL #18994

**Comment by zy-kkk** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

File: docs/en/docs/lakehouse/external-table/jdbc.md [hunk 1]
```diff
@@ -248,17 +248,27 @@ properties (
     "password"="",
     "jdbc_url" = "jdbc:oceanbase://localhost:2881/test",
     "driver_url" = "file:///path/to/oceanbase-client-2.4.2.jar",
-    "driver_class" = "com.oceanbase.jdbc.Driverr"
+    "driver_class" = "com.oceanbase.jdbc.Driver",
+    "oceanbase_mode" = "oceanbase" or "oceanbase_oracle"
```
Comment (morningman) [inline docs/en/docs/lakehouse/external-table/jdbc.md:252]:
```suggestion
    "oceanbase_mode" = "mysq" or "oracle"
```

**Review by BePPPower — DISMISSED** [review]

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by luzhijing — APPROVED** [review]

**Comment by zy-kkk** [comment]
run buildall

**Pull request merged**